wormhole-connect/vite.config.ts

```diff
@@ -37,6 +40,8 @@ const resolve = {
     hooks: path.resolve(__dirname, './src/hooks'),
     consts: path.resolve(__dirname, './src/consts'),
     public: path.resolve(__dirname, './public'),
+    'process/': 'process',
+    'buffer/': 'buffer',
```

Comment:
this is required to un-break one of our downstream dependencies that imports these with a trailing slash...